<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>416</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>539</width>
    <height>416</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>539</width>
    <height>416</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MoYu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="moyuTitle">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MoYu</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idInputBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="sendMsgButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>280</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="outputBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>491</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>391</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
